planner/optimizer/optimizer-core/src/main/java/eu/seaclouds/platform/planner/optimizer/Reoptimizer.java

```diff
@@ -54,7 +54,7 @@ public Reoptimizer(SearchMethodName name) {
       }
    }
 
-   public String[] optimize(String appModel, String suitableCloudOffer,
+   public String[] optimize(String appModel, String suitableCloudOffer, String benchmarkPlatforms,
          int numPlansToGenerate_ArrayLength, double hyst) {
       // TODO Auto-generated method stub
       return null;
```

Comment:
@perezp If this functionality is not yet implemented why do you want to push it to the master? Maybe is better to leave in a separate branch or even in a stash until you develop it. In any case if the return type is an array, at least return an empty one to avoid NPE.